pkg/action/scan.go

```diff
@@ -18,6 +19,12 @@ import (
 	"github.com/hillu/go-yara/v4"
 )
 
+var (
```

Comment:
Since we already have the root path if we are working with an archive from here:
https://github.com/chainguard-dev/bincapz/blob/main/pkg/action/archive.go#L236

I'm wondering if we could use that to strip instead of regexp? Maybe we could add that into the Config and use it in scanSinglePath 🤔

Reply:
Oh, yea. That's a good point.

Reply:
Changed in [`e76e4bc` (#217)](https://github.com/chainguard-dev/bincapz/pull/217/commits/e76e4bc1f043925dc4d198a87864b56204786cbc).